web/templates/stream_settings/change_stream_info_modal.hbs

```diff
@@ -3,6 +3,7 @@
         {{t 'Channel name' }}
     </label>
     <input type="text" id="change_stream_name" class="modal_text_input" name="stream_name" value="{{ stream_name }}" maxlength="{{ max_stream_name_length }}" />
+    <div id="stream_name_error"></div>
```

Comment:
There will be two elements with same ID due to this as same ID is use in stream creation form as well. Either you can use a class here or a different ID like `change_stream_name_error`.